src/static/css/link.css

```diff
@@ -0,0 +1,42 @@
+.link-container {
+    padding: 2rem;
+    width: min-content;
+}
+
+a {
+    color: var(--fl-linkColor, var(--link-color));
+    text-decoration: underline;
+}
+
+a:active {
+    background: var(--fl-bgColor, var(--link-active-color-bg));
+    box-shadow: 0 0 0 0.3em var(--fl-bgColor, var(--link-active-color-bg));
+    color: var(--fl-linkColor, var(--link-active-color));
+}
+
+a:focus {
+    color: var(--fl-linkColor, var(--link-focus-color));
+    outline: 3px solid var(--fl-linkColor, var(--link-focus-color));
+    outline-offset: 6px;
+    outline-width: 3px;
+}
+
+a:hover {
+    color: var(--fl-linkColor, var(--link-hover-color));
+}
```

Comment:
We should use the order indicated here: https://css-tricks.com/css-basics-styling-links-like-boss/#aa-style-each-link-state